yaml/thz504.yaml

```diff
@@ -4,4 +4,4 @@ esphome:
       - "-DTHZ_504"
 
 packages:
-  BASE: !include { file: wp_base.yaml}
+  BASE: !include { file: wp_base.yaml}
```

Comment:
👎

Reply:
This I do not get fully. I can not remove commits from git history. If you do not want to take the history with the revert my only idea would be to create a new branch & pull request?

Reply:
You are on your dev branch so everything is allowed ;)
first do a 
> git pull origin --rebase
so that my commit vanishes from your branch

> git rebase -i HEAD~8
the remainign 8 commits can partially be dropped 

> drop move includes into main yaml
drop adding pipethrough of pin config to wp_base
pick addind eco 5.5 specifics
squash fixed indentation
sqash added guard
pick using substitutions as pin config
drop Revert "move includes into main yaml"
drop Merge branch 'master' into adapted-for-thz-5.5eco

should look the same on code view, but has way better git history

Reply:
Thanks! Cleanup did work as you described.